File: plugboard.js
const alphabet = 'abcdefghijklmnopqrstuvwxyz';

class Plugboard {
  constructor(changes = []) {
    this.mainDict = {};

    for (const letter of alphabet) {
      this.mainDict[letter] = letter;
    }

    for (const pair of changes) {
      this.mainDict[pair[0]] = pair[1];
      this.mainDict[pair[1]] = pair[0];
    }
  }

  pass(letter) {
    return this.mainDict[letter];
  }
}

module.exports = Plugboard;

File: plugboard.py
import string

alphabet = string.ascii_lowercase

class Plugboard:

    def __init__(self, changes=[]):
        self.main_dict = dict(zip(alphabet, alphabet))
        for pair in changes:
            self.main_dict[pair[0]] = pair[1]
            self.main_dict[pair[1]] = pair[0]

    def pass_(self, letter):
        return self.main_dict[letter]